### PR TITLE
Fix for FHIR-48927 to remove Location identifier search parameter from AU Core

### PR DESCRIPTION
**Pull request by dt-r**
This PR fixes [FHIR-48927](https://jira.hl7.org/browse/FHIR-48927) Remove identifier search parameter from Location. Changes to the following:
* Change log
* Location notes page
* AU Core Requester CapabilityStatement
* AU Core Responder CapabilityStatement

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This change log documents the significant updates and resolutions implemented fr
   - Changed the Location 'address' search parameter from SHALL to SHOULD [FHIR-47107](https://jira.hl7.org/browse/FHIR-47107)
   - Added a PractitionerRole 'role' search parameter as a SHOULD [FHIR-47115](https://jira.hl7.org/browse/FHIR-47115)
   - Changed MedicationRequest search parameters 'patient+intent' and 'patient+intent+status' from SHALL to SHOULD, and added a new MedicationRequest search parameter 'patient+status' as a SHALL [FHIR-47073](https://jira.hl7.org/browse/FHIR-47073)
+  - removed Location search parameter 'identifier' [FHIR-48927](https://jira.hl7.org/browse/FHIR-46209)
```
Comment (JayMurdoch) [inline input/pagecontent/changes.md:28]: Change Jira url to 48927

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -37,6 +38,7 @@ This change log documents the significant updates and resolutions implemented fr
...
```
Comment (JayMurdoch) [inline input/pagecontent/changes.md:41]: Change Jira url to 48927

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Two minor changes required, thanks.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: fixed

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: fixed

**Comment by dt-r** [comment]
> Two minor changes required, thanks.

In addition, changed the intro notes page to remove the comparison to other IGs content - this change was missed in a previous PR.

File: input/intro-notes/StructureDefinition-au-core-location-intro.md [hunk 3]
```diff
@@ -6,16 +6,6 @@ The following are supported usage scenarios for this profile:
 - Read information about a location referenced by another resource
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-location-intro.md:6]: Commenting here to get an inline comment. 

Missing a change to add a sentence to intro, above the Usage Scenarios, to link to the Comparison page that is present in other profiles pages.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Confirmed.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Confirmed.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One minor change requested.

**Comment by dt-r** [comment]
> One minor change requested.

fixed.

File: input/intro-notes/StructureDefinition-au-core-location-intro.md (hunk 3 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-location-intro.md:8]: fixed

**Pull request merged**